samples/Mvc.Client/Startup.cs

```diff
@@ -49,6 +49,11 @@ public class Startup {
                 options.ClientSecret = "98f1bf028608901e9df91d64ee61536fe562064b";
             });
 
+            app.UseLinkedInAuthentication(options => {
```

Comment:
I guess you can't easily re-generate your client_id/client_secret via LinkedIn?
I'll try to create a register a new app to test it :sake: